### PR TITLE
🔨 Event Fix

### PR DESCRIPTION
**Pull request by JaredBorders**

### CODE REVIEW

**Review by JChiaramonte7 — APPROVED** [review]

**Comment by JChiaramonte7** [comment]
Actually going to un-approve this one. @JaredBorders can you add a test that verifies if you close a position. A fee is collected and an event is emitted.

**Comment by JaredBorders** [comment]
Test verifying fee collected already exists: https://github.com/Kwenta/margin-manager/blob/beb39a32d0103226e68d396c3985fb996832a970/test/integration/margin.behavior.ts#L1318
Adding Test for event in following commit (and updating test comment which is incorrect now)

**Comment by JaredBorders** [comment]
@JChiaramonte7 ready for review

**Comment by JaredBorders** [comment]
Maybe let Diana know this change was made?

File: test/integration/margin.behavior.ts [hunk 1]
```diff
@@ -1373,6 +1369,64 @@ describe("Integration: Test Cross Margin", () => {
...
+                .distributeMargin(newPosition);
+
+            let receipt: ContractReceipt = await tx.wait();
+            const event = receipt.events?.filter((x) => {
+                return x.event == "FeeImposed";
+            });
+            const accountAddr = event![0].args![0];
+            const feeImposedAmount = event![0].args![1];
+            expect(accountAddr).to.equal(marginAccount.address);
+            expect(feeImposedAmount).to.equal(fee);
```
Comment (JChiaramonte7) [inline test/integration/margin.behavior.ts:1427]: Just so you know for next time there is a built-in where you can do `await expect(tx).to.be.emit(contract, "EVENT_NAME").withArgs(address, fee, etc...)`

https://github.com/Kwenta/margin-manager/blob/beb39a32d0103226e68d396c3985fb996832a970/test/integration/order.behavior.ts#L216-L218

**Review by JChiaramonte7 — APPROVED** [review]

**Pull request merged**